src/families/filecoin/bridge/account.ts

```diff
@@ -116,29 +116,27 @@ const estimateMaxSpendable = async ({
   const a = getMainAccount(account, parentAccount);
   const { address } = getAddress(a);
 
+  const recipient = transaction?.recipient;
+
   if (!validateAddress(address).isValid) throw new InvalidAddress();
```

Comment:
Unrelated to this PR but I just noticed, probably no need to validate the user's own account address since it's from the Live itself?...